docs/testing/regression-checklist/mapping.json

```
@@ -13,20 +13,20 @@
 		"testFile": [ "extensions/all.md" ]
 	},
 	{
-		"regex": "extensions/blocks/contact-form*",
+		"regex": "extensions/blocks/contact-form.*",
 		"testFile": [ "extensions/blocks/contact-form.md" ]
 	},
 
 	{
-		"regex": "modules/publicize*",
+		"regex": "modules/publicize.*",
 		"testFile": [ "modules/publicize.md" ]
 	},
 	{
 		"regex": "modules/stats.php",
 		"testFile": [ "modules/stats.md" ]
 	},
 	{
-		"regex": "extensions/blocks/*",
+		"regex": "extensions/blocks/(?!CHANGELOG|changelog/)",
```

There shouldn't ever be a changelog inside `extensions/blocks/`. The full path is `projects/plugins/jetpack/extensions/blocks/` (or `projects/plugins/jetpack/tests/php/extensions/blocks/`), and the corresponding changelog entries would be in `projects/plugins/jetpack/`.
```suggestion
		"regex": "extensions/blocks/",
```